src/mixins/textfield-mixin.js

```diff
@@ -0,0 +1,63 @@
+export default {
+  props: {
+    value: [String, Number],
+    nativeValue: [String, Number],
+
+    type: String,
+
+    id: String,
+    disabled: Boolean,
+    required: Boolean,
+    name: String,
+    label: {
+      type: String,
+      default: null,
+    },
+    placeholder: String,
+
+    errorMessage: String,
+
+    error: Boolean,
+    clearable: Boolean,
+    ghost: Boolean,
+    withIconLeft: Boolean,
```

Comment:
What do you think we have a property called `icon` (`String` name with the icon) that always place the icon on left, and another property `rightIcon` (`String` name with the icon), that places the icon on right? I believe it's more readable and consistency to the other devs to use this component. And, I believe it's not necessary to have a property icon that expected an object with size.